tools/c7n_azure/c7n_azure/entry.py

```diff
@@ -17,6 +17,11 @@
 import c7n_azure.resources.storage
 import c7n_azure.resources.resourcegroup
 import c7n_azure.resources.sqlserver
+import c7n_azure.resources.load_balancer
```

Comment:
(minor thing) Maybe we can start sorting this alphabetically. It keeps it nice and organized.